src/applications/vaos/actions/expressCare.js

```diff
@@ -115,6 +124,54 @@ export function fetchExpressCareWindows() {
   };
 }
 
+/**
+ * Fetches request limits for all active windows and selects the first one
+ * where the user has not reached their request limit
+ */
+export function fetchRequestLimits() {
+  return async (dispatch, getState) => {
+    dispatch({
+      type: FORM_FETCH_REQUEST_LIMITS,
+    });
+
+    try {
+      const activeFacilityIds = selectActiveExpressCareWindows(
+        getState(),
+        moment(),
+      ).map(win => win.facilityId);
+
+      // Temporarily limit concurrent calls to 5 while we
+      // wait for a new endpoint that will accept multiple facilityIds
+      const requestLimits = await Promise.all(
+        activeFacilityIds
+          .slice(0, 5)
+          .map(facilityId => getRequestLimits(facilityId, EXPRESS_CARE)),
+      );
```

Comment:
Without the loop and splice, wouldn't this only make the request limit call for the first 5 facilityIds?

Reply:
Yes, which I think is ok for a while.